src/copa_game/server.clj

```diff
@@ -37,4 +36,7 @@
 (comment
   (build-system-map)
   (start-system! (build-system-map))
-  (stop-system!))
\ No newline at end of file
+  (stop-system!)
+  (def http (:http @system-atom))
+  (get-teams http)
+  )
```

Comment:
I don't think the http things should be here. If you'd like to have a file where you can play with different parts of the system, I think a file in the namespace `dev.user` (/dev/user.clj) is more common.